### PR TITLE
Feature/fixing validation

### PR DESCRIPTION
**Pull request by klngwll**
First step in fixing some of our validation errors in marc properties. See each commit for details.

See [LXL-3421](https://jira.kb.se/browse/LXL-3421)

### CODE REVIEW

File: source/marc/enums.ttl [hunk 1]
```diff
@@ -1603,13 +1603,13 @@
...
-    skos:notation "VisualRunningTimeType" ;
-    skos:prefLabel "Running Time"@en,
-        "Speltid"@sv .
+# :VisualRunningTimeType a :CollectionClass ;
+#     :category :pending ;
+#     rdfs:subClassOf :EnumeratedTerm ;
+#     skos:broadMatch kbv:Visual ;
+#     skos:notation "VisualRunningTimeType" ;
+#     skos:prefLabel "Running Time"@en,
+#         "Speltid"@sv .
```
Comment (olovy) [inline source/marc/enums.ttl:1612]: Sure we don't want to just erase it?

**Review by olovy — APPROVED** [review]
LGTM but I think it needs some more :eyes:

File: source/marc/enums.ttl (hunk 1 above)
Comment (klngwll) [reply source/marc/enums.ttl:1612]: Well we have taken the cautious path with these, since this one never had been used it might be less of a worry.

**Comment by lrosenstrom** [comment]
I trust Fredrik more than myself on this. Looks reasonable!

**Review by lrosenstrom — APPROVED** [review]

**Pull request merged**